myia/compile/backends/__init__.py

```diff
@@ -36,7 +36,7 @@ def loader():
 }
 
 # This is used as the default backend and options if None is provided
-default_name = 'nnvm'
+default_name = 'pytorch'
```

Comment:
Is this intentional?  Not that I'm against it necessarily.

Reply:
It was to get rid of the annoying warnings due to the `pytest.ini` change, but they pop up anyway, so that's probably pointless.